README.md

```diff
@@ -1,9 +1,9 @@
-# Nix Installer
+# The Determinate Nix Installer
 
 [![Crates.io](https://img.shields.io/crates/v/nix-installer)](https://crates.io/crates/nix-installer)
 [![Docs.rs](https://img.shields.io/docsrs/nix-installer)](https://docs.rs/nix-installer/latest/nix_installer/)
 
-`nix-installer` is an opinionated, **experimental** Nix installer.
+The Determinate `nix-installer` is an opinionated, **experimental** Nix installer.
```

Comment:
Mmm, I actually don't think this is necessary, since we're referring to the binary at this point, no?

If we do want this, we probably want it to be `The Determinate Nix Installer`, and this should be changed below under `## Status` as well (and any other places)...

Reply:
Whoops